glide.yaml

```diff
@@ -25,6 +25,7 @@ import:
   - libmachine/state
   - libmachine/swarm
 - package: github.com/google/go-github
+  version: 30a21ee1a3839fb4a408efe331f226b73faac379
```

Comment:
> which seems to urgent and blocking from new users to build it properly.

There is no issue building the project from scratch using our build. The problem lies in trying to build it from Fedora rpms. However, this approach needs to stop looking at glide.yaml, but rather look at glide.lock. Unless the exact pinned down versioned are used, whatever the build results, it is not Minishift.